Resources/doc/reference/architecture.rst

```diff
@@ -5,7 +5,7 @@ The architecture of the ``SonataAdminBundle`` is primarily inspired by the Djang
 Project, which is truly a great project. More information can be found at the
 `Django Project Website`_.
 
-If you followed the instructions on the :doc:`getting_started` page, you should by
+If you followed the instructions on the :doc:`../getting_started/creating_an_admin` page, you should by
```

Comment:
Here is what I would say normally : please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

But it does not seem to 72 chars here , so either adapt to what it currently is, or rewrap everything to 72 chars.